brainvisa/toolboxes/tools/processes/assemblynet.py

```diff
@@ -99,14 +101,19 @@ def execution(self, context):
     )
 
     for file_path in output_folder.iterdir():
-        if file_path.is_dir():
+        if file_path.is_dir() or file_path.suffix == '.minf':
             continue
-        # if file_path.name.startswith("mni") or file_path.name.startswith("matrix"):
         if file_path.name.startswith("mni") or file_path.name.startswith("native"):
             signature_name = '_'.join(file_path.name.split('_')[:2])
             output_file = getattr(self, signature_name).fullPath()
-            if not Path(output_file).exists():
-                file_path.rename(output_file)
+            if Path(output_file).suffix == '.gz':
+                if not Path(output_file).exists():
+                    file_path.rename(output_file)
```

Comment:
I see that those line already exists before, but it may be an issue if the output_file already exists from another run of assemblyNet, and we want to replace old run with new results?

Reply:
Yes you are right I have remove those `if` and do the rename always.